mp/src/game/server/cbase.cpp

```diff
@@ -642,10 +642,10 @@ DEFINE_FIXEDSIZE_ALLOCATOR( EventQueuePrioritizedEvent_t, 128, CUtlMemoryPool::G
 
 CEventQueue g_EventQueue;
 
-CEventQueue::CEventQueue()
+CEventQueue::CEventQueue() : m_iRealTickCount(0)
```

Comment:
This is the only instance where m_iRealTick gets set to 0. There should be a reset function so that players can change maps and be able to fire boosters without an undefined delay.